services/QuillLMS/app/services/classlink_integration/auth_credential_saver.rb

```diff
@@ -17,7 +17,7 @@ def initialize(code:, action_type:)
     def run
       generate_auth_credential
     rescue => e
-      ErrorNotifier.report(RunError.new, { message: e.to_s })
+      ErrorNotifier.report(RunError.new, { message: "#{e}\n#{e.backtrace}" })
```

Comment:
This is a pattern I want to start using, for exceptions that we rescue but may want to peek into, for diagnostic purposes